public/app/features/dashboard/components/GenAI/GenAIButton.tsx

```diff
@@ -29,10 +36,10 @@ export const GenAIButton = ({
 
   const replyHandler = (response: string, isDone: boolean) => {
     setLoading(!isDone);
-    onReply(response, isDone);
+    onGenerate(response, isDone);
   };
 
-  const onGenerate = (e: React.MouseEvent<HTMLButtonElement>) => {
+  const clickHandler = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
Nit: not following frontend style guide: https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#props